src/shared/orgTypes.ts

```diff
@@ -52,7 +52,6 @@ export type FullyPopulatedScratchOrgFields = ScratchOrgFields &
 
 // developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_scratchorginfo.htm
 export type ScratchOrgInfoSObject = {
-  Id: string;
   CreatedDate: string;
```

Comment:
I don't see this `Id` field either in docs or the API response. The org ID returned by `org display` is comes from the auth file.